randovania/data/json_data/prime1/Phazon Mines.txt

```diff
@@ -873,13 +889,17 @@ Asset id: 340985721
       Any of the following:
           Space Jump Boots
           Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner) and Phazon Damage ≥ 150
+          X-Ray Visor and Combat/Scan Dash (Beginner) and L-Jump (Beginner) and Phazon Damage ≥ 150
+          Phazon Suit and Slope Jump (Intermediate)
+          Slope Jump (Advanced) and Phazon Damage ≥ 300
```

Comment:
Why does having Phazon Suit make this an easier slope jump? Why not make it the same difficulty so its either phazon suit or damage?

Reply:
Removed the intermediate slope jump. It's pretty tight and it's annoying to learn/perform while taking phazon damage.